src/test/java/POM/FactoryPages/LoginPage.java

```diff
@@ -0,0 +1,44 @@
+package POM.FactoryPages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class LoginPage extends BasePage {
+
+    @FindBy(css = "[type = 'email']")
+    WebElement emailField;
+    @FindBy(css = "[type = 'password']")
+    WebElement passwordField;
+    @FindBy(css = "[type = 'submit']")
+    WebElement submitBtn;
+
+    public LoginPage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+    public LoginPage clickSubmitBtn() {
+
+        submitBtn.click();
+        return this;
+    }
+
+    public LoginPage provideEmail(String email) {
+
+        emailField.sendKeys(email);
+        return this;
+    }
+
+    public LoginPage providePassword(String password) {
+
+        passwordField.sendKeys(password);
+        return this;
+    }
+
+    public void login() {
+        driver.manage().window().maximize();
```

Comment:
Please, move `driver.manage().window().maximize();` to `class BaseTest`

Base Test in the main class which will take care of Browser setup, loading configuration file and other reusable methods like screenshot, handling sync issues and many more.

![image](https://user-images.githubusercontent.com/104761482/203918191-df06c4ba-65f0-4831-b210-278a23d08359.png)